app/controllers/api/v1/section_content_controller.rb

```diff
@@ -1,9 +1,16 @@
 module Api
   module V1
     class SectionContentController < ApiController
+      DEFAULT_LANGUAGE = 'en'.freeze
+
       def index
         locale = params[:locale]
-        section_contents = locale ? SectionContent.where(locale: locale) : SectionContent.all
+        section_contents =
+          if locale
```

Comment:
could be simplified with:

`SectionContent.where(locale: params[:locale].presence || DEFAULT_LANGUAGE)`

if you like one liners :p

Reply:
might not even need the `presence` to be sure. oh... I just thought that maybe we should constrain to "en", "id", have a sort of dictionary of available languages? What you think?

Reply:
💪 done